de-lib/src/main/java/org/iplantc/de/theme/base/client/apps/cells/AppRatingCellDefaultAppearance.java

```diff
@@ -181,6 +188,13 @@ public void render(SafeHtmlBuilder sb, App app) {
         int total = app.getRating().getTotal();
         // Build five rating stars
         for (int i = 0; i < ratings.size(); i++) {
+            if (!app.isPublic() || app.getAppType().equalsIgnoreCase(App.EXTERNAL_APP)) {
+                sb.append(templates.imgCell("Rating-" + i,
+                                            ratings.get(i),
+                                            resources.css().disabledRating(),
+                                            resources.whiteStar().getSafeUri()));
```

Comment:
I guess that's up for debate.  IMO users would be more confused that there are no stars and think it's a bug or think their private apps could never be rated versus having a tooltip that tells them why they can't currently rate those apps.  And then with external apps we let them know it's just not supported.